src/main/java/com/facebook/openwifirrm/modules/RRMScheduler.java

```diff
@@ -332,8 +332,10 @@ protected void performRRM(String zone) {
 				modeler,
 				zone,
 				params.dryRun,
-				true /* allowDefaultMode */
+				true, /* allowDefaultMode */
+				false /* updateImmediately */
 			);
+			configManager.queueZoneAndWakeUp(zone);
```

Comment:
do we want to do it within the loop? seems better to queue it once and wake it up after all the algos have run, especially since put `updateImmediately` as false.